content/en/blog/_posts/2025-01-15-swap-fresh-improvements.md

```diff
@@ -0,0 +1,403 @@
+---
+layout: blog
+title: "Kubernetes 1.32: Fresh Swap Features for Linux Users"
+date: 2025-01-15T10:00:00-08:00
+slug: swap-linux-improvements
+author: >
+  Itamar Holder (Red Hat)
```

Comment:
If you add `draft: true` here, we can merge this as a draft and then sort out the publication date later.
(I suggest you do that).

Reply:
Done!